app/posts/register-for-an-npq/2023-06-01-registration-funding-status.md

```diff
@@ -39,6 +47,36 @@ After a round of research and more in depth conversations with the development t
 
 [See screenshots further down this page](#1.-funding-status-1)
 
+## 2. Provider details 
+
+### The problem
+- Users did not know what 'pending' meant, as a status under 'Provider application'.
+- Users had different interpretations of what 'pending' meant. 
+- Users wanted to know what else needed to be done or if it was out of their hands. 
+
+Pending can mean a variety of things and was being used when we didn't have a definite alternate status e.g. accepted or rejected. We are unable to get a status for every point on the journey e.g. application submitted. This leaves the status with too many potential options and unknowns and doesn't give the user clarity on what to do next. 
+
+### Design changes 
+
+1. Provider status has been removed throughout to avoid confusion for users. We often would not have an update to providde to users, so the status could be inaccurate. 
```

Comment:
```suggestion
1. Provider status has been removed throughout to avoid confusion for users. We often would not have an update to provide to users, so the status could be inaccurate. 
```